code/weapon/weapon.h

```diff
@@ -38,6 +38,15 @@ extern int Num_weapon_subtypes;
 #define	WRT_LASER	1
 #define	WRT_POF		2
 
+// constants for weapon lock acquire methods
+#define WLOCK_PIXEL		0
+#define WLOCK_TIMER		1
+
+// constants for weapon lock restrictions
+#define LR_CURRENT_TARGET				0		// Only lock current target and subsystem
+#define LR_CURRENT_TARGET_SUBSYS		1		// 
+#define LR_ANY_TARGETS					2
```

Comment:
You could use constexpr here instead of define as all compilers we use, support that.

Reply:
I'd like further discussion on any particular reason to change that here instead of passing over many of our defines in one go.

Reply:
Yeah, I don't have a problem with deferring that and then making a separate PR to change things to `constexpr`.